$NetBSD: patch-ac,v 1.1 2011/01/03 07:58:43 dsainty Exp $

Include <cstring> to declare memset(), used from this header in inlines.

--- Imath/ImathMatrix.h.orig	2010-07-17 10:48:40.000000000 +1200
+++ Imath/ImathMatrix.h	2011-01-03 20:30:27.543078712 +1300
@@ -49,6 +49,7 @@
 #include "ImathVec.h"
 #include "ImathShear.h"
 
+#include <cstring>
 #include <iostream>
 #include <iomanip>
 
